apps/web/src/components/team-builder/v2/builder.module.css

```
@@ -1056,35 +1101,68 @@
   font-size: 8.5px;
   font-weight: 700;
   letter-spacing: 0.12em;
   text-transform: uppercase;
   color: var(--muted-foreground);
-  padding: 0 8px 5px;
-  height: 21px;
-  display: flex;
-  align-items: flex-end;
 }
 
 :global(.calc-col-row) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)